Example/Storage/Tests/Unit/FIRStorageListTests.m

```diff
@@ -55,16 +55,32 @@ - (void)tearDown {
 }
 
 - (void)testValidatesInput {
+  XCTestExpectation *expectation = [self expectationWithDescription:@"testValidatesInput"];
+
   FIRStoragePath *path = [FIRStorageTestHelpers objectPath];
   FIRStorageReference *ref = [[FIRStorageReference alloc] initWithStorage:self.storage path:path];
 
-  FIRStorageVoidListError emptyBlock = ^(FIRStorageListResult *result, NSError *error) {
+  __block int errorCount = 0;
+
+  FIRStorageVoidListError errorBlock = ^(FIRStorageListResult *result, NSError *error) {
+    XCTAssertNil(result);
+    XCTAssertNotNil(error);
+
+    XCTAssertEqualObjects(error.domain, @"FIRStorageErrorDomain");
+    XCTAssertEqual(error.code, FIRStorageErrorCodeInvalidArgument);
+
+    ++errorCount;
+    if (errorCount == 4) {
```

Comment:
You may use [expectedFulfillmentCount](https://developer.apple.com/documentation/xctest/xctestexpectation/2806572-expectedfulfillmentcount?language=objc) instead.

Reply:
Very nice :) Thanks